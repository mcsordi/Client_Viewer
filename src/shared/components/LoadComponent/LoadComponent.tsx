import React from 'react';

export const LoadComponent: React.FC = () => {
  return (
    <div className="relative flex justify-center items-center min-w-full bg-yellow-400 h-2 rounded-md">
      <div className="absolute animate-ping  w-6/12 rounded-full">
        <div className="relative bg-orange-800 h-1 rounded-md "></div>
      </div>
    </div>
  );
};
