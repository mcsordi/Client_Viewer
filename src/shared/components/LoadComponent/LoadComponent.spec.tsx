import { render } from '@testing-library/react';
import { LoadComponent } from './LoadComponent';

describe('<LoadComponent/>', () => {
  it('should render a LoadComponent', () => {
    const { container } = render(<LoadComponent />);
    expect(container.firstChild).toHaveClass('bg-yellow-400');
    expect(container.firstChild.firstChild).toHaveClass('animate-ping');
    expect(container.firstChild.firstChild.firstChild).toHaveClass(
      'bg-orange-800',
    );
  });
  it('should match a snapshot', () => {
    const { container } = render(<LoadComponent />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
