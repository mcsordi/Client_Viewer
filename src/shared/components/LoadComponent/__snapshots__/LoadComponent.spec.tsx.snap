// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoadComponent/> should match a snapshot 1`] = `
<div
  class="relative flex justify-center items-center min-w-full bg-yellow-400 h-2 rounded-md"
>
  <div
    class="absolute animate-ping  w-6/12 rounded-full"
  >
    <div
      class="relative bg-orange-800 h-1 rounded-md "
    />
  </div>
</div>
`;
